scripts/release_prep.sh

```diff
@@ -6,12 +6,9 @@ else
     JLAB_REL_BRANCH=$1
     JLAB_REL_ENV=jlabrelease_$JLAB_REL_BRANCH
 
-    WORK_DIR=$(mktemp -d)
+    WORK_DIR=$(mktemp -d -t $JLAB_REL_ENV)
     cd $WORK_DIR
 
-    conda deactivate
```

Comment:
Why don't we need this here?  To me it looks like we should instead be adding this to `release_test.sh` as well to make sure we start from a fresh env.

Reply:
conda activate appropriately nests environments these days.

Reply:
i.e.,

```
conda activate A
conda activate B
```
puts you in environment B, and `conda deactivate` puts you back in A.

I'm always running these scripts from a fresh terminal anyway. If you're not, there's no guarantee that `conda deactivate` actually deactivates all environments.

Reply:
I meant the remove part.  I needed to call deactivate first because it wouldn't remove the active env.  But point taken about using a fresh terminal.

Reply:
conda now automatically removes an environment if you create one with the same name as an existing one: https://github.com/conda/conda/pull/7133